packages/eas-cli/src/build/utils/url.ts

```diff
@@ -4,20 +4,24 @@ import { getExpoApiBaseUrl, getExpoWebsiteBaseUrl } from '../../api';
 import { AppPlatform, BuildFragment } from '../../graphql/generated';
 
 export function getProjectDashboardUrl(accountName: string, projectName: string): string {
-  return `${getExpoWebsiteBaseUrl()}/accounts/${accountName}/projects/${projectName}`;
+  return new URL(
+    `/accounts/${accountName}/projects/${projectName}`,
+    getExpoWebsiteBaseUrl()
+  ).toString();
 }
 
 export function getBuildLogsUrl(build: BuildFragment): string {
   const { project } = build;
+  let url = `/builds/${build.id}`;
```

Comment:
We try to avoid using `let` if it's not necessary. In this case, you can get away with not using `let` really easily.